pkg/kubectl/genericclioptions/print_flags.go

```diff
@@ -35,7 +38,8 @@ func (e NoCompatiblePrinterError) Error() string {
 		output = *e.OutputFormat
 	}
 
-	return fmt.Sprintf("unable to match a printer suitable for the output format %q and the options specified: %#v", output, e.Options)
+	sort.Strings(e.AllowedFormats)
```

Comment:
@soltysh 
Since I use `range map[key]bool` to iterate the map.
The order of output would be uncertain. So I add a sort here.

Reply:
I don't think I care about this, but ok.